backend/app/modules/searchengine/SearchActor.scala

```diff
@@ -3,16 +3,13 @@ package modules.searchengine
 import java.security.InvalidParameterException
 
 import akka.actor.{Actor, ActorRef, Props}
-import modules.scheduler.SchedulerActor
 import akka.pattern.ask
 import akka.util.Timeout
+import models.Query
+import modules.scheduler.MonitoringActor
 
 import scala.concurrent.Await
-import scala.concurrent.Future
-import scala.concurrent.duration.Duration
-import scala.concurrent.duration._
-
-import models.Process
+import scala.concurrent.duration.{Duration, _}
```

Comment:
What's the point of importing *Duration* and *everything* afterward? Isn't Duration part of everything?